### ISSUE TITLE
DataType Precision Loss on model creation.

### ISSUE THREAD

**Issue by mheil23** [sequelize/sequelize-auto#160]
I'm generating a model from a postgres database and I'm having a pretty significant issue. There are database constraints that have been placed on the columns in the database for specific reasons (validations, data integrity, etc..). However, when sequelize-auto generates the model, it doesn't include these constraints. For example, a postgres column = **first_name VARCHAR(50) NOT NULL.** This column comes through as
 `
first_name: {
type: DataTypes.STRING,
allowNull: false
}
`
It completely loses the length constraint. Is there a work around for this?

**Comment by durango** [comment]
Not saying that this isn't an issue nor am I saying I wont attempt to fix this down the road when I have the time but for now I thought you should know that adding constraints is more efficient for length checking on Postgres than MySQL.

```sql
ALTER TABLE names
  ADD CONSTRAINT name_constraint CHECK (char_length(name) <= 255);
```

**Comment by steveschmitt** [comment]
I think this is fixed in `sequelize-auto@0.5.0`
Please try it and close the issue if it is fixed.

**Comment by steveschmitt** [comment]
Fixed in `sequelize-auto@0.5.1`.
